### DISCUSSION TITLE
Icons are not displayed

### DISCUSSION THREAD

**Discussion by mario81k** [Blackymas/NSPanel_HA_Blueprint#557] [Q&A] [answered]
Hello everyone, First of all thanks for the great work. I'm totally excited about the blueprint, but I have a problem with the current version and can't get any further. Neither chips nor other buttons (e.g. the WLAN QR code) are displayed on the homepage. Can someone give me a tip here?
[image](https://user-images.githubusercontent.com/121456551/228321525-c4073a6f-21ad-4032-ad23-33ae454adbf8.jpeg)

**Comment by Blackymas** [comment, marked as answer]
hi,

I think you use a old TFT version.

On the bnotpage you will see your versions

**Reply by mario81k** [reply]
Hello Blackmas,
sorry just got around to checking it out. Although I updated all versions today, the TFT version is 3.1.0, ESP Home 3.2.2.
How can I bring the version up to date?
I just got the following error message when updating the ESP:

12:21:54 | [W] | [next:396] | Nextion reported variable name invalid!

Can this be related?
Thanks for the support.
Mario

**Reply by edwardtfn** [reply]
> Hello Blackmas, sorry just got around to checking it out. Although I updated all versions today, the TFT version is 3.1.0, ESP Home 3.2.2. How can I bring the version up to date? I just got the following error message when updating the ESP:
> 
> 12:21:54 | [W] | [next:396] | Nextion reported variable name invalid!
> 
> Can this be related? Thanks for the support. Mario

Yes, this is related, so when you update the TFT some of the error messages will disappear.

If you have set up your ESPHome file properly, then you just need to access your panel in Home Assistant under Settings > Devices and Services > ESPHome and click the Button "Update TFT", then the upload process will start and if everything goes well in a few minutes you will have the new TFT running and icons should be visible.
If that fails, take a look if you have followed all the process. There's a lot about this in the docs and other posts like [this one](https://github.com/Blackymas/NSPanel_HA_Blueprint/issues/619#issuecomment-1499770066).

**Reply by edwardtfn** [reply]
Hey @mario81k, have you solved this issue? Should we close this discussion?
